scripts/copyuser/main.go

```diff
@@ -0,0 +1,1504 @@
+
+	err := c.WithTx(func(tx *sql.Tx) error {
+		q := c.SQLBuilder.Select("id").From("_auth_user").Where("app_id = ?", c.FromAppID)
+
+		err := c.Query(tx, q, func(rows *sql.Rows) error {
+			var userID string
+			err := rows.Scan(&userID)
+			if err != nil {
+				return err
+			}
+			userIDs = append(userIDs, userID)
+			return nil
+		})
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return userIDs, nil
+}
+
+func (c *Copier) CopyUser(userID string) error {
+	err := c.WithTx(func(tx *sql.Tx) error {
+		fmt.Printf("copying user %v to %v\n", userID, c.NewID(userID))
+
+		err := c.CopyUserTable(tx, userID)
+		if err != nil {
+			return err
+		}
+
+		err = c.CopyRecoveryCodeTable(tx, userID)
+		if err != nil {
+			return err
+		}
+
+		err = c.CopyPasswordHistoryTable(tx, userID)
+		if err != nil {
+			return err
+		}
+
+		err = c.CopyOAuthAuthorizationTable(tx, userID)
+		if err != nil {
+			return err
+		}
+
+		err = c.CopyVerifiedClaimTable(tx, userID)
+		if err != nil {
+			return err
+		}
+
+		err = c.CopyIdentity(tx, userID)
+		if err != nil {
+			return err
+		}
+
+		err = c.CopyAuthenticator(tx, userID)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+
+	return err
+}
+
+func (c *Copier) CopyUserTable(tx *sql.Tx, userID string) error {
+	columns := []string{
+		"id",
+		"app_id",
+		"created_at",
+		"updated_at",
+		"last_login_at",
+		"login_at",
+		"is_disabled",
+		"disable_reason",
+		"standard_attributes",
+		"custom_attributes",
+		"is_deactivated",
+		"delete_at",
+		"is_anonymized",
+		"anonymize_at",
+		"anonymized_at",
+	}
+	tableName := "_auth_user"
+
+	q := c.SQLBuilder.Select(columns...).
+		From(tableName).
+		Where("app_id = ? AND id = ?", c.FromAppID, userID)
+
+	var id string
+	var appID string
+	var created_at time.Time
+	var updated_at time.Time
+	var last_login_at sql.NullTime
+	var login_at sql.NullTime
+	var is_disabled bool
+	var disable_reason sql.NullString
+	var standard_attributes []byte
+	var custom_attributes []byte
+	var is_deactivated sql.NullBool
+	var delete_at sql.NullTime
+	var is_anonymized bool
+	var anonymize_at sql.NullTime
+	var anonymized_at sql.NullTime
+
+	err := c.QueryRow(tx, q, func(row *sql.Row) error {
+		return row.Scan(
+			&id,
+			&appID,
+			&created_at,
+			&updated_at,
+			&last_login_at,
+			&login_at,
+			&is_disabled,
+			&disable_reason,
+			&standard_attributes,
+			&custom_attributes,
+			&is_deactivated,
+			&delete_at,
+			&is_anonymized,
+			&anonymize_at,
+			&anonymized_at,
+		)
+	})
+	if err != nil {
+		return err
+	}
+
+	insert := c.SQLBuilder.Insert(tableName).Columns(columns...).Values(
+		c.NewID(id),
+		c.ToAppID,
+		created_at,
+		updated_at,
+		last_login_at,
+		login_at,
+		is_disabled,
+		disable_reason,
+		standard_attributes,
+		custom_attributes,
+		is_deactivated,
+		delete_at,
+		is_anonymized,
+		anonymize_at,
+		anonymized_at,
+	).Suffix(onConflictDoNothing)
```

Comment:
To allow re-running of the script, we should skip copying all the other table entries for that user if the user exists. 

For the scenario,

1. Run the script to copy users from app A to app B
2. User in app A delete and add an identity
3. Run the script again
4. The user will have 2 identities

To keep it simple, maybe we can support only copying the newly added user and skip copying all the other table entries?